cmd/cli/main.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"encoding/json"
+
+	"github.com/segmentio/analytics-go"
+	"github.com/segmentio/conf"
+)
+
+func main() {
+	var config struct {
+		WriteKey   string `conf:"writeKey"   help:"The Segment Write Key of the project to send data to"`
+		Type       string `conf:"type"       help:"The type of the message to send"`
+		UserID     string `conf:"userId"     help:"Unique identifier for the user"`
+		GroupID    string `conf:"groupId"    help:"Unique identifier for the group"`
+		Traits     string `conf:"traits"     help:"Metadata associated with the user"`
+		Event      string `conf:"event"      help:"Name of the track event"`
+		Properties string `conf:"properties" help:"Metadata associated with an event, page or screen call"`
```

Comment:
The `conf` package supports map types, why not use that so it gets loaded automatically instead of getting a string and parsing it?

Reply:
Didn't know it did that, updating!

Reply:
Turns out the objconv pacakge doesn't handle the `interface{}` in `map[string]interface{}` well. unfortunately we need this so we can exercise the library with a wider range of fixtures, so we're sticking with manual parsing for now. I improved the error message in case of a JSON parsing failure though!

Reply:
Actually objconv can deal with this just fine, it’s conf that has trouble with it ;)

:+1: